ws-server/src/tests.rs

```diff
@@ -57,8 +57,8 @@ async fn server() -> SocketAddr {
 /// It has the following methods:
 ///     sync methods: `say_hello` and `add`
 ///     async: `say_hello_async` and `add_sync`
-///     other: `invalid_params` (always returns `CallError::InvalidParams`), `call_fail` (always returns `CallError::Failed`), `sleep_for`
-/// Returns the address together with handles for server future and server stop.
+///     other: `invalid_params` (always returns `CallError::InvalidParams`), `call_fail` (always returns
+/// `CallError::Failed`), `sleep_for` Returns the address together with handles for server future and server stop.
```

Comment:
I guess if this were a proper markdown list it would indent properly!

Reply:
Yeah this is a bit annoying.